### PR TITLE
Added olm 4.3 data

### PR DESCRIPTION
**Pull request by zvonkok**
Adding OLM 4.3 data now with all missing file, last commit was incomplete.

### CODE REVIEW

**Comment by zvonkok** [comment]
@sjug New commit, PTAL

File: Makefile [hunk 1]
```diff
@@ -54,6 +54,7 @@ deploy-crds: $(DEPLOY_CRDS)
 deploy-objects: deploy-crds
 	for obj in $(DEPLOY_OBJECTS); do \
 		$(TEMPLATE_CMD) $$obj | kubectl apply -f - ;\
```
Comment (sjug) [inline Makefile]: Why `kubectl` rather than `oc`?

File: Makefile [hunk 2]
```diff
@@ -54,6 +54,7 @@ deploy-crds: $(DEPLOY_CRDS)
...
+		sleep 1;\
```
Comment (sjug) [inline Makefile]: I realize it's simple enough to work, but alternatively maybe do a wait?

**Comment by sjug** [comment]
Is there any validation we can do for the 4th commit, 4.3 olm data?

File: Makefile (hunk 1 above)
Comment (zvonkok) [reply Makefile]: For deployments on Kubernetes

File: Makefile (hunk 2 above)
Comment (zvonkok) [reply Makefile]: This is only used for manual deployment when developing and testing, the main install path is per  OLM. Do not want to put a lot of "logic" into this.

**Comment by zvonkok** [comment]
@sjug This is the same OLM data based on 4.2 just exchanged the version. You could create a package with operator-courier with the right images and additionally create a CatalogSource to inject this repository into your running cluster.

**Pull request merged**